### PR TITLE
Main

### PR DESCRIPTION
**Pull request by SvartAlfe**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by SvartAlfe** [comment]
ok

**Comment by SvartAlfe** [comment]
@microsoft-github-policy-service agree

File: .env [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FLASK_DEBUG=True
+DBNAME=team-a-hackathon-database
+DBHOST=team-a-hackathon-server.postgres.database.azure.com
+DBUSER=dwvpwrjszj
+DBPASS=NXVNJB117212N2T3
+SECRET_KEY=4c3c228c3a464df54a7fc1a5919bc025497cba06654e921596e41484f96321c1
```
Comment (pamelafox) [inline .env:6]: You have checked your secret key in! Please revoke it immediately.

File: app.py [hunk 2]
```diff
@@ -31,93 +33,103 @@
...
+                InsertQnA(question, embeddings, answer)
+        return 'success' + logs
+    except Exception as e:
+        return 'error again in qnainit ' + str(e)
+
+def do_completion(prompt):
+    logs = "logs:"
+
+    try:
+        openai.api_key = "efa40032bc644d449c3bf80610e21228"
```
Comment (pamelafox) [inline app.py:101]: You have checked this key in a public PR, please revoke immediately!

**Comment by pamelafox** [comment]
This looks like a private PR that was unintentionally made public, and it includes secrets. Please be very careful with your secrets and revoke the exposed ones immediately, for your sake. Closing this PR since it seems like it was meant to be private.

**Comment by bobtabor-msft** [comment]
Closing per @pamelafox 's request.

**Pull request closed without merging**